### DISCUSSION TITLE
Does oxlint support for extends from preset?

### DISCUSSION THREAD

**Discussion by Orenbek** [oxc-project/oxc#6548] [Q&A]
is it possible for oxlint to support for `extends` settings in configuration file?
for instance:
```json
{
"extends": ["plugin:react/recommended"]
}
```
why this is necessary? In order to migrate from an existing eslint configured project to oxlint, we wanna collect our existing eslint rules into a preset, so that our team could directly use it in their projects.

**Comment by Boshen** [comment]
https://oxc.rs/docs/guide/usage/linter/nested-config.html
